homeassistant/components/ios/mobile_app.py

```diff
@@ -0,0 +1,33 @@
+"""iOS app specific logic."""
+import logging
+from aiohttp.web import json_response
+
+from homeassistant.helpers import template
+
+from . import CONF_PUSH, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+WEBHOOK_TYPE_GET_PUSH_CONFIG = 'get_push_config'
+WEBHOOK_TYPE_RENDER_COMPLICATIONS = 'render_complications'
+
+WEBHOOK_TYPES = [WEBHOOK_TYPE_GET_PUSH_CONFIG,
+                 WEBHOOK_TYPE_RENDER_COMPLICATIONS]
+
+
+async def async_handle_webhook_message(hass, device, webhook_type, data):
+    """Return a webhook response for the given arguments."""
+    _LOGGER.debug("iOS: received webhook %s %s %s", webhook_type, device,
+                  data)
+
+    if webhook_type == WEBHOOK_TYPE_GET_PUSH_CONFIG:
+        return json_response(hass.data[DOMAIN][CONF_PUSH])
+
+    if webhook_type == WEBHOOK_TYPE_RENDER_COMPLICATIONS:
```

Comment:
I am wondering if we should just support this in the normal template rendering webhook: render multiple templates at once. Make the input a dictionary of key -> template, validate it with `{str: cv.template}`. You can then just use [`render_complex`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/template.py#L53) which will go through the dictionary and render it all.

Reply:
Holding this for a new PR